cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -155,6 +155,10 @@ public BuildCommand() {
     @CommandLine.Option(names = "--cloud", description = "Enable cloud artifact generation")
     private String cloud;
 
+    @CommandLine.Option(names = "--show-dependency-diagnostics", description = "Show the diagnostics " +
+            "generated by the dependencies")
```

Comment:
The new command option name `--show-dependency-diagnostics` could imply that errors are also being suppressed if it's not enabled. So, I'm thinking whether `--show-dependency-warnings` would be a clearer choice. @azinneera thoughts?

Reply:
It could imply the warnings only right... we need suppress info, hint as well..

Reply:
We discussed this fact earlier @NipunaRanasinghe.  We are showing hints, info, and warnings at the moment. But there is plan to suppress errors in the future as well: hide actual errors show a user-friendly version. If a user wants to see actual errors, this option needs to be given. 

With that, I think `--show-dependency-diagnostics` is good.

Reply:
@sameerajayasoma got it :) yeah that makes sense